layouts/inibling/menu.html

```diff
@@ -24,6 +24,8 @@
 			  </ul>
 			</li>
 			<li><a href="/atomic">Atomic</a></li>
+			<li><a href="/erstifahrt">Ersti-Fahrt</a></li>
+			<li><a href="/iniyourface">Ini Your Face</a></li>
```

Comment:
Naja, das ist ja in einem commit der nur die Menüpunkte hinzugefügt, ist schon ok so